gmm/handler.go

```diff
@@ -1530,6 +1530,56 @@ func AuthenticationProcedure(ue *context.AmfUe, accessType models.AccessType) (b
 	return false, nil
 }
 
+func NetworkInitiatedDeregistrationProcedure(ue *context.AmfUe,
+	accessType models.AccessType) (err error) {
+
+	anType := util.AnTypeToNas(accessType)
+	if ue.CmConnect(accessType) {
+		//setting reregistration required flag to true
+		gmm_message.SendDeregistrationRequest(ue.RanUe[accessType], anType, true, 0)
+	} else {
+		SetDeregisteredState(ue, anType)
+	}
+	// TODO: Need to implement Nudm_SDM_Unsubscribe
```

Comment:
Check if AMF is sending callback URL to UDM, only then AMF need to send Unsubscribe to UDM.

Reply:
yes, its required .. created new bug for unsubscribe handling in AMF and UDM